src/components/login/LoginForm/PasswordInput/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { passwordInputStyle, errorTextStyle } from "./style";
+
+export default function PasswordInput({
+  onValidityChange,
+}: {
+  onValidityChange: (valid: boolean) => void;
+}) {
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const validatePassword = (value: string) => {
+    if (!value) return "PW를 입력해주세요.";
+    if (value.length < 8) return "PW는 최소 8글자 이상이어야 합니다.";
```

Comment:
if문을 한 줄로 처리하는 것을 지양해주세요. 가능하다면 블록으로 명확히 구조화하는 걸 권장드립니다!
```suggestion
    if (!value) {
      return "PW를 입력해주세요.";
    }
    if (value.length < 8) {
      return "PW는 최소 8글자 이상이어야 합니다.";
    }
```